### ISSUE TITLE
 undefined: balancer.PickOptions

### ISSUE THREAD

**Issue by CocoaZX** [bilibili/kratos#502]
![企业微信截图_158078624622](https://user-images.githubusercontent.com/28552051/73710840-0e342980-4740-11ea-9d26-f6772c374a88.png)

项目创建成功.
PS E:\codeIDE\code\XiLife_Goods> cd kratos-demo/cmd
PS E:\codeIDE\code\XiLife_Goods\kratos-demo\cmd> go build
# github.com/bilibili/kratos/pkg/net/rpc/warden/balancer/p2c
E:\codeIDE\go\goPath\pkg\mod\github.com\bilibili\kratos@v0.3.3\pkg\net\rpc\warden\balancer\p2c\p2c.go:158:52: undefined: balancer.PickOptions
E:\codeIDE\go\goPath\pkg\mod\github.com\bilibili\kratos@v0.3.3\pkg\net\rpc\warden\balancer\p2c\p2c.go:190:52: undefined: balancer.PickOptions
# github.com/bilibili/kratos/pkg/net/rpc/warden/resolver
E:\codeIDE\go\goPath\pkg\mod\github.com\bilibili\kratos@v0.3.3\pkg\net\rpc\warden\resolver\resolver.go:53:78: undefined: resolver.BuildOption
E:\codeIDE\go\goPath\pkg\mod\github.com\bilibili\kratos@v0.3.3\pkg\net\rpc\warden\resolver\resolver.go:110:33: undefined: resolver.ResolveNowOption

**Comment by solarhell** [comment]
https://github.com/etcd-io/etcd/issues/11563
降级grpc即可解决。

**Comment by CocoaZX** [comment]
先装protobuf，在new工程不会出错了

**Comment by cjb21sj** [comment]
强制绑定grpc 1.26.0
```
module kratos-demo

go 1.13

require (
	github.com/bilibili/kratos v0.3.4-0.20200306133340-7c946dc919c1
	github.com/gogo/protobuf v1.3.1
	github.com/golang/protobuf v1.3.5
	github.com/google/wire v0.4.0
	github.com/pkg/errors v0.9.1 // indirect
	google.golang.org/genproto v0.0.0-20200318110522-7735f76e9fa5
	google.golang.org/grpc v1.27.0
)

replace (
	github.com/coreos/go-systemd => github.com/coreos/go-systemd/v22 v22.0.0
	google.golang.org/grpc => google.golang.org/grpc v1.26.0
)
```

**Comment by modood** [comment]
这个问题是 grpc 版本升级不兼容导致的，可以像 cjb21sj 所说的一样使用 v1.26.0，也可以参考以下链接替换成新的 API ：[clientv3: Fix grpc-go(v1.27.0) incompatible changes to balancer/resolver. #11564](https://github.com/etcd-io/etcd/pull/11564/files)
